R/chop.R

```diff
@@ -181,18 +187,26 @@ df_unchop <- function(x, ..., ptype = NULL, keep_empty = FALSE, error_call = cal
       next
     }
 
+    col <- tidyr_new_list(col)
+    col_sizes <- list_sizes(col)
+    col_nulls <- vec_detect_missing(col)
```

Comment:
Slightly awkward to compute the `NULL` locations here and again in `list_replace_null()`, but I'm convinced this is probably due to the `df_unchop()` algorithm being slightly suboptimal, and I didn't want to mess with `list_replace_null()` just for this one usage.